examples/plot_example_knockoffs.py

```diff
@@ -0,0 +1,138 @@
+"""
+Controlled variable selection: why and how?
+=======================================
+"""
+# In this example, we explore the basics of variable selection and illustrate the need to statistically control the amount of faslely selected variables. We start off by loading the Wisconsin breast cancer dataset:
+
+#%%
+import numpy as np
+
+# Set random seed for reproducibility
+seed = 99
+np.random.seed(seed)
+
+# Load and standardize the breast cancer dataset
+from sklearn.datasets import load_breast_cancer
+from sklearn.preprocessing import StandardScaler
+
+data = load_breast_cancer()
+X = data.data
+y = data.target
+
+X = StandardScaler().fit_transform(X)
+
+print(X.shape)
+n, p = X.shape
+
+
```

Comment:
Add a new section to make the text appear in a text block and note a code comment. 
```suggestion

#############################################################################
```